### DISCUSSION TITLE
Legal - Intellectual Property

### DISCUSSION THREAD

**Discussion by rastaman111** [TortugaPower/BookPlayer#682] [Q&A]
Hello @GianniCarlo!

My question is: how did you bypass Apple's verification that your application uses media from third-party sources?

**Comment by pichfl** [comment]
Like any other app that manages media files (File managers, music players, photo editors, and many more) we only organize and present user provided files. We strongly belief in a DRM free economy, and so do many publishers and outlets which provide audiobooks without encryption.

We do not include and will never provide any content material inside our application.

**Reply by rastaman111** [reply]
Did I understand you correctly.  Have you proved to the app that you have no right to restrict the user from downloading media files from third-party sites and listening to them in your application?

**Reply by pichfl** [reply]
Given our [privacy policy](https://github.com/TortugaPower/BookPlayer/blob/main/PRIVACY_POLICY.md), we do not have access to *any* content users store in our app. 

I'm not sure I understand your comment / question. When a user loads a file into BookPlayer, it is their file on their device inside their copy of BookPlayer. There is no service or storage we control in any way. We are provide a tool to organize and consume content, not a content provider.

**Reply by pichfl** [reply]
For context: [Section 5.2.3 of the App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/#intellectual-property)

> Audio/Video Downloading: Apps should not facilitate illegal file sharing or include the ability to save, convert, or download media from third-party sources (e.g. Apple Music, YouTube, SoundCloud, Vimeo, etc.) without explicit authorization from those sources. Streaming of audio/video content may also violate Terms of Use, so be sure to check before your app accesses those services. Documentation must be provided upon request.

We _do not_ support any form of download from third-party sources which require explicit authorization. You _can_ download from public http links (as provided by sources like [LibriVox](https://librivox.org) which have the necessary licenses. 

The distinction here is the same as mentioned above, we do not interact with any third party service directly to obtain content. We use official file providers for which a user has to authenticate (like Dropbox, iCloud, Google Drive, …)

**Comment by rastaman111** [comment]
This is what Apple writes to me when checking the application:

Guideline 5.2.3 - Legal - Intellectual Property


Your app allows users to save or download music, video, or other media content without authorization from the relevant third-party sources.

The next submission of this app may require a longer review time, and this app will not be eligible for an expedited review until this issue is resolved.

**Reply by pichfl** [reply]
I think this thread is going from being about BookPlayer to developer support for your own application. We can only help to a certain extend there. 

Does your app promote or prominently feature direct downloads from third party resources? We do offer a url schema, but it's mostly aimed at publishers who rely on BookPlayer as their preferred player solution.

**Reply by rastaman111** [reply]
I have a similar function in the application like yours.
Uploaded a media file from a computer or other cloud storage to the application and listen to it as long as you like
